step registry integration: nil pointer

Seen in an [integration job run](https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_ci-tools/281/pull-ci-openshift-ci-tools-master-integration/983):

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x13cec21]
goroutine 118 [running]:
github.com/openshift/ci-tools/pkg/load.Registry.func1(0xc00039e2c0, 0x3f, 0x0, 0x0, 0x19840e0, 0xc0003cee40, 0x0, 0x0)
	/go/src/github.com/openshift/ci-tools/pkg/load/load.go:93 +0x81
path/filepath.walk(0xc000402270, 0x28, 0x19cdf00, 0xc0002049c0, 0xc000373cf8, 0x0, 0x0)
	/usr/local/go/src/path/filepath/path.go:378 +0x20c
path/filepath.walk(0xc000043440, 0x20, 0x19cdf00, 0xc000204680, 0xc000373cf8, 0x0, 0x0)
	/usr/local/go/src/path/filepath/path.go:382 +0x2ff
path/filepath.walk(0x7ffcfddbc352, 0x1c, 0x19cdf00, 0xc0002045b0, 0xc000373cf8, 0x0, 0xc000373cc0)
	/usr/local/go/src/path/filepath/path.go:382 +0x2ff
path/filepath.Walk(0x7ffcfddbc352, 0x1c, 0xc000373cf8, 0x153df64b408bb, 0xc021590a06)
	/usr/local/go/src/path/filepath/path.go:404 +0xff
github.com/openshift/ci-tools/pkg/load.Registry(0x7ffcfddbc352, 0x1c, 0x24a9800, 0xc0003cecf0, 0xc0003ced20, 0xc0003ced50, 0x0, 0x0)
	/go/src/github.com/openshift/ci-tools/pkg/load/load.go:92 +0xfe
github.com/openshift/ci-tools/pkg/load.(*registryAgent).loadRegistry(0xc000151220, 0x0, 0x0)
	/go/src/github.com/openshift/ci-tools/pkg/load/registryAgent.go:118 +0xdc
github.com/openshift/ci-tools/pkg/coalescer.(*coalescer).Run.func1()
	/go/src/github.com/openshift/ci-tools/pkg/coalescer/coalescer.go:40 +0x6c
sync.(*Once).doSlow(0xc000442850, 0xc0004ef760)
	/usr/local/go/src/sync/once.go:66 +0xe3
sync.(*Once).Do(...)
	/usr/local/go/src/sync/once.go:57
github.com/openshift/ci-tools/pkg/coalescer.(*coalescer).Run(0xc0002e4ca0, 0x0, 0x0)
	/go/src/github.com/openshift/ci-tools/pkg/coalescer/coalescer.go:34 +0xb8
github.com/openshift/ci-tools/pkg/load.reloadWatcher.func1(0x1982a40, 0xc0002e4ca0)
	/go/src/github.com/openshift/ci-tools/pkg/load/agent_utils.go:42 +0x35
created by github.com/openshift/ci-tools/pkg/load.reloadWatcher
	/go/src/github.com/openshift/ci-tools/pkg/load/agent_utils.go:41 +0x32e 
```

/assign @AlexNPavel

--- a/pkg/load/load.go
+++ b/pkg/load/load.go
@@ -90,7 +90,7 @@ func Registry(root string, flat bool) (references map[string]api.LiteralTestStep
 	chains = map[string][]api.TestStep{}
 	workflows = map[string]api.MultiStageTestConfiguration{}
 	err = filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
-		if strings.HasPrefix(info.Name(), "..") {
+		if info != nil && strings.HasPrefix(info.Name(), "..") {
 			if info.IsDir() {
 				return filepath.SkipDir
 			}

